src/services/form-validation/validate-dot-js.ts

```diff
@@ -107,7 +95,7 @@ export function isEmpty(value: unknown) {
 
   // If we find at least one property we consider it non empty
   if (isObject(value)) {
-    for (attr in value) {
+    for (const _attr in value) {
```

Comment:
why did you need to _ it if you removed the let?
```suggestion
    for (const attr in value) {
```

Reply:
eslint like variables that not used to be prefixed with `_`